Program.cs

```diff
@@ -1,57 +1,396 @@
+                            }
+                        } while (!cityStats.CityIsValid(response));
+                        cityStats.DisplayCityInfo(response);
+                        Console.WriteLine("Do you want to return to the Mode Selection Menu? (y/n):");
+                        if (Console.ReadKey().KeyChar == 'y')
+                            valid = true;
+                        break;
+                    case "2":
+                        Console.Write("\nPlease key in a province name to return the city with the largest population in that given province: ");
+                        do
+                        {
+                            response = Console.ReadLine();
+                            response = FormatAnswer(response);
+                            if (!cityStats.ProvinceIsValid(response))
+                            {
+                                Console.Write("\nInvalid response. Please pick from one of the following cities and key in the city name: ");
+                                cityStats.DisplayCitiesList();
+                                Console.WriteLine("Please write your selection from the above list: ");
+                            }
+                        } while (!cityStats.ProvinceIsValid(response));
+                        cityStats.DisplayLargestPopulationCity(response);
+                        Console.Write("Do you want to return to the Mode Selection Menu? (y/n):");
+                        if (Console.ReadKey().KeyChar == 'y')
+                            valid = true;
+                        break;
+                    case "3":
+                        Console.Write("\nPlease key in a province name to return the city with the smallest population in that given province: ");
+                        do
+                        {
+                            response = Console.ReadLine();
+                            response = FormatAnswer(response);
+                            if (!cityStats.ProvinceIsValid(response))
+                            {
+                                Console.Write("\nInvalid response. Please pick from one of the following cities and key in the city name: ");
+                                cityStats.DisplayCitiesList();
+                                Console.WriteLine("Please write your selection from the above list: ");
+                            }
+                        } while (!cityStats.ProvinceIsValid(response));
+                        cityStats.DisplaySmallestPopulationCity(response);
+                        Console.Write("Do you want to return to the Mode Selection Menu? (y/n): ");
+                        if (Console.ReadKey().KeyChar == 'y')
+                            valid = true;
+                        break;
+                    case "4":
+                        Console.Write("\nPlease key in two cities, separated by ',' to compare the population of those two cities: ");
+                        do
```

Comment:
let's maybe just separate this out into two inputs. Keeps it simple and avoids it breaking if they put a space after the comma as most people would